test/remote/config_update_tests.js

```diff
@@ -56,7 +56,7 @@ test(
       })
       .then(function (bis) {
         x = bis
-        return mcHelper.setLimits({ blockIntervalSeconds: bis + 1, uidRateLimit:{} })
+        return mcHelper.setLimits({ blockIntervalSeconds: bis + 1 })
```

Comment:
This appears to have been a hack around this problem, it can be removed now that the nested object is handled correctly.